src/mqt/predictor/rl/helper.py

```diff
@@ -393,7 +395,10 @@ def get_cmap_from_devicename(device: str):
 
 
 def create_feature_dict(qc):
-    feature_dict = {"num_qubits": qc.num_qubits, "depth": qc.depth()}
+    feature_dict = {
+        "num_qubits": np.array([qc.num_qubits], dtype=int),
+        "depth": np.array([qc.depth()], dtype=int),
```

Comment:
Because the Open AI Gym and Stable-Baselines are really picky about the format the observation space is defined, especially for the new representation using a `dict` with more suitable data types for each feature vector element. Before, we used a simple construct. More details can be found here, for example: https://stackoverflow.com/questions/73922332/dict-observation-space-for-stable-baselines3-not-working

Reply:
Thanks for the link. Kind of strange that the tool is so restrictive, but if that's how it goes, then so be it 👍🏼